README.md

```diff
@@ -83,21 +83,19 @@ Using [Composer](https://getcomposer.org/), you can install multiform into your
 SilverStripe site using this command (while in the directory where your site is
 currently located)
 
-	composer require "silverstripe/multiform:*"
+`composer require "silverstripe/multiform:*"`
```

Comment:
This would render `like this inline` rather than in a code block - maybe use triple backticks on the lines around it